pkg/controllers/disruption/orchestration/queue.go

```diff
@@ -154,12 +158,15 @@ func NewCommand(replacements []string, candidates []*state.StateNode, id types.U
 }
 
 func (q *Queue) Register(_ context.Context, m manager.Manager) error {
-	return controller.NewSingletonManagedBy(m).
+	return controllerruntime.NewControllerManagedBy(m).
 		Named("disruption.queue").
-		Complete(q)
+		WatchesRawSource(singleton.Source()).
+		Complete(singleton.AsReconciler(q))
 }
 
-func (q *Queue) Reconcile(ctx context.Context, _ reconcile.Request) (reconcile.Result, error) {
+func (q *Queue) Reconcile(ctx context.Context) (reconcile.Result, error) {
+	ctx = injection.WithControllerName(ctx, "disruption.queue")
```

Comment:
should we rename the folder? Or match the injection to the package? 
```suggestion
	ctx = injection.WithControllerName(ctx, "orchestration.queue")
```

Reply:
Same comment here. I think we should decouple this into a separate PR so that we aren't making additional changes in the context of this refactor.